backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/JurisXmlExporterResponseProcessor.java

```diff
@@ -178,6 +183,9 @@ private Optional<MessageWrapper> forwardMessage(MessageWrapper messageWrapper) {
                 return messageWrapper;
               });
 
+    } catch (DocumentationUnitNotExistsException e) {
+      //      LOGGER.info("Could not forward Message: {}", messageWrapper, e);
+      return Optional.empty();
```

Comment:
Hier ist für uns zu überlassen, ob wir überhaupt in diesem Fall es als info weiter geben wollen. Da ist die Frage, warum haben wir Dokumente von juris die nicht bei uns liegen wie XXRE000000224, und gegeben ist den Fall, müssen wir es an Juris weiter leiten oder wie gehen wir damit.

Reply:
this is completly our side. we send a documentation unit to juris and get back an answer mail if the documentation unit is right. this message we attach to the documentation unit. I think we generate a documentation unit and published it. after that we delete the documentation unit in our system. We should think about what's the right way to handle this. But in another ticket!